### PR TITLE
Support Anthropology Major

### PR DESCRIPTION
**Pull request by Destaq**
### Summary 

Rite of passage — added support for A&S's anthropology major. This involved updating `index.ts` and adding `anthr.ts` in the `src/data` folder. Also updated the requirements JSON file as required.

You can find major requirements for this degree over [here](https://courses.cornell.edu/preview_program.php?catoid=55&poid=28110).

### Test Plan 

Updated the huge JSON file and requirements generator. All tests passed. Behold the new Anthropology major visually as well:

image

I also tried to add various classes on localhost but it didn't seem to persist / was a bit buggy. However, the same problem was happening for my existing data (e.g. adding new CS major classes) so I'm not sure if it's possible to test this manually like that.

### Notes 

The anthropology major is easier to obtain than I thought.

### Blockers 

N/A

### Breaking Changes  

N/A

### CODE REVIEW

File: src/data/majors/anthr.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  courseMeetsCreditMinimum,
+  courseIsFWS,
+} from '../../requirements/checkers';
+import { AdvisorGroup } from '../../tools/advisors/types';
+
+const anthrRequirements: readonly CollegeOrMajorRequirement[] = [
+  {
+    name: 'Sociocultural, Archaeological, and Biological Anthropology',
+    description:
+      'One course of 3 or more credits in each of the three subfields (sociocultural, archaeological, biological) from the list below.' +
```
Comment (elizabeth-tang) [inline src/data/majors/anthr.ts:16]: Do we need to check if these courses are >=3 credits?

File: src/data/majors/anthr.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        const { catalogNbr, subject } = course;
+        return (
+          ifCodeMatch(subject, 'ANTHR') &&
+          ifCodeMatch(catalogNbr, '4***') &&
+          courseMeetsCreditMinimum(course, 3)
+        );
+      },
+    ],
+    checkerWarning:
+      'We do not check a course is of seminar style with a research paper / project component.',
```
Comment (elizabeth-tang) [inline src/data/majors/anthr.ts:74]: lol small grammar thing 'We do not check if a course is of seminar style with a research paper / project component.'

**Review by elizabeth-tang — CHANGES_REQUESTED** [review]
Looks good! Just a minor suggestion and question for you! Welcome to the team!!!

File: src/data/majors/anthr.ts (hunk 1 above)
Comment (Destaq) [reply src/data/majors/anthr.ts:16]: I mean they give you a fixed list of applicable courses, so I thought not. But might as well add in.

File: src/data/majors/anthr.ts (hunk 2 above)
Comment (Destaq) [reply src/data/majors/anthr.ts:74]: I knew the sentence sounded sus, will get on fixing it 👍

File: src/data/majors/anthr.ts (hunk 1 above)
Comment (Destaq) [reply src/data/majors/anthr.ts:16]: Yeah so I looked around the other majors and never saw this being combined with a `minCredit` check. So gonna defer on this tell I get confirmation / cancellation.

**Review by elizabeth-tang — APPROVED** [review]

**Comment by Destaq** [comment]
Should I squash and merge?

**Pull request merged**